evap/results/tests/test_views.py

```diff
@@ -1,12 +1,40 @@
-from django_webtest import WebTest
-from django.test import TestCase
+from model_mommy import mommy
 
-from evap.evaluation.models import Semester
-from evap.results.exporters import ExcelExporter
+from evap.evaluation.models import Semester, UserProfile
+from evap.evaluation.tests.test_utils import ViewTest
 
-class UsecaseTests(WebTest):
+
+class TestResultsView(ViewTest):
+    url = '/results/'
+    test_users = ['evap']
+
+    @classmethod
+    def setUpTestData(cls):
+        mommy.make(UserProfile, username='evap')
+
+
+class TestResultsSemesterDetailView(ViewTest):
+    test_users = ['evap']
+
+    @classmethod
+    def setUpTestData(cls):
+        mommy.make(UserProfile, username='evap')
+
+        semester = mommy.make(Semester)
+        cls.url = '/results/semester/%s' % semester.id
+
+
+class TestResultsSemesterCourseDetailView(ViewTest):
+    url = '/results/semester/1/course/8'
+    test_users = ['contributor', 'evap', 'responsible']
     fixtures = ['minimal_test_data_results']
 
+    def test_single_result_course(self):
+        url = '/results/semester/1/course/11'
```

Comment:
why are there 11 courses, the cls.url is 8, this tests 11, and the other tests 1?